### PR TITLE
Finished poc to: 1. write create a stream from random model 2. stream to mysql sink

### PR DESCRIPTION
**Pull request by stancsz**
generated streams for courier model to be written to the database

console log:
image

data generated in the database:
image

### CODE REVIEW

**Comment by stancsz** [comment]
@DoTrongAnh you can pull my code from my branch by checking it out:
```
git remote add stans-branch git@github.com:stancsz/scala-data-stream-test.git
git fetch stans-branch

git checkout -b new-master stans-branch/master

```

File: src/main/scala/AppActivityGenerator/RandomSource.scala [hunk 1]
```diff
@@ -45,29 +45,34 @@ object RandomSource {
...
   }
 
-  def generateOrder = new Order
+  def generateOrder = {
+    val order = Seq(new Order)
+    val done =
+      Source(order)
+        .log("order")
+        .runWith(
+          Slick.sink(order => sqlu"INSERT INTO DeliveryDB.OrderTest (order_id,order_score,app_created_timestamp,lat,lon) VALUES(${order.order_id}, ${order.order_score}, ${order.order_created}, ${order.lat}, ${order.lon})")
```
Comment (DoTrongAnh) [inline src/main/scala/AppActivityGenerator/RandomSource.scala:70]: Naming specific columns only if the schema is expected to change during stream.

File: src/main/scala/AppActivityGenerator/RandomSource.scala [hunk 2]
```diff
@@ -61,7 +52,7 @@ object RandomSource {
       Source(cour)
         .log("cour")
         .runWith(
-          Slick.sink(cour => sqlu"INSERT INTO DeliveryDB.CourierTest VALUES(${cour.courier_id}, ${cour.courier_score}, ${cour.courier_created}, ${cour.lat}, ${cour.lon})")
+          Slick.sink(cour => sqlu"INSERT INTO DeliveryDB.CourierTest (courier_id,courier_score,app_created_timestamp,lat,lon) VALUES(${cour.courier_id}, ${cour.courier_score}, ${cour.courier_created}, ${cour.lat}, ${cour.lon})")
```
Comment (DoTrongAnh) [inline src/main/scala/AppActivityGenerator/RandomSource.scala:60]: Should be 'INSERT IGNORE' since courier_id is also specified in the tuple.

**Review by DoTrongAnh — COMMENTED** [review]
Some comments on the sql statements.

**Pull request merged**